modules/evm/src/bench/mock.rs

```diff
@@ -188,7 +187,7 @@ impl Config for Runtime {
 	type PrecompilesValue = ();
 	type ChainId = ConstU64<1>;
 	type GasToWeight = GasToWeight;
-	type ChargeTransactionPayment = DefaultTransactionPayment;
+	type ChargeTransactionPayment = DefaultTransactionPayment<Balances>;
```

Comment:
We don't need to implement this. It's not used

Reply:
any reason why? maybe move evm bench to runtime/mandala/benchmarking will make things(I mean use relay impl which depend on transaction payment module) a lot easier?

Reply:
we don't have to maintain mirror this implementation

Reply:
Add module_dex, and use it's real implementation.
do we need to re-benchmark?